tests/update-file-to-s3.ts

```diff
@@ -1,7 +1,9 @@
 import { mockAxios } from './helpers/mock-axios';
 jest.mock('axios', () => ({ default: mockAxios }));
-
+/* tslint:disable */
```

Comment:
I don't think this should be necessary, and defo better with:

```ts
// tslint:disable-next-line:import-order
```

To prevent other rules being ignored, and the need to re-enable tslint.

Will pop over to your desk.

Reply:
Please do, it didn't work quite well for me!